public/app/features/logs/components/log-context/LogRowContextModal.tsx

```diff
@@ -388,11 +388,21 @@ export const LogRowContextModal: React.FunctionComponent<LogRowContextModalProps
           <Button
             variant="secondary"
             onClick={async () => {
+              let rowId = row.uid;
+              if (row.dataFrame.refId) {
+                rowId = row.uid.replace(row.dataFrame.refId, contextQuery.refId);
```

Comment:
It took me a second to figure out why are we doing this, so maybe adding a comment would make sense here.

Reply:
So true! Added in b621944b475918a1e69f3402ab59acf985f72d0e